pkg/storage/storagebackend/factory/tls_test.go

```diff
@@ -34,7 +34,8 @@ import (
 )
 
 func TestTLSConnection(t *testing.T) {
-	certFile, keyFile, caFile := configureTLSCerts(t)
+	tempDir, certFile, keyFile, caFile := configureTLSCerts(t)
+	defer os.RemoveAll(tempDir)
```

Comment:
Actually, `configureTLSCerts` will nerver return a empty tempDir, because the test case will fail directly when running `ioutil.TempDir` fail in function `configureTLSCerts`, and `os.RemoveAll(tempDir)` will run well even with the `tempDir` is empty. I'm not sure if this can dispel your nervous. :)